library/general/src/lib/config_file/config_file.rb

```diff
@@ -0,0 +1,29 @@
+module ConfigFile
+  class ConfigFile
+    def initialize(parser, file_path, file_class: File)
+      @file_class = file_class
+      @parser = parser
+      @file_path = file_path
+    end
+
+    def write(changes_only: false)
```

Comment:
So `read`, `write` are part of the public API... Wouldn't it be better to adopt an ActiveModel API, with `save` instead?